packages/manager/apps/web/client/app/hosting/hosting.controller.js

```diff
@@ -265,7 +266,13 @@ export default class {
         this.$scope.hosting.sqlPriveInfo.nbDataBaseInclude = this.$scope.hosting.offerCapabilities.privateDatabases.length;
         this.$scope.hosting.sqlPriveInfo.nbDataBaseActive =
           this.$scope.hosting.sqlPriveInfo.nbDataBaseInclude -
-          privateDbCapabilities.length;
+          reduce(
```

Comment:
why don't use `Array.reduce` instead of `lodash/reduce`?

Reply:
What about https://lodash.com/docs/4.17.15#sumBy ?

Reply:
We can keep it as is since it's has been validated.

But you're right, we have multiple way to address this fix.

Thanks for your suggestions!